README.md

```diff
@@ -39,3 +43,7 @@ The current evaluation template is a Jupyter notebook using Bokeh for interactiv
 ```
 jupyter notebook
 ```
+
+However the scheme for running on notebook server [mybinder.org](https://mybinder.org) is a little more involved and uses `bokeh serve`. You can create an interactive bokeh app server on mybinder by following the link below.
+
+[`https://mybinder.org/v2/gh/riveSunder/carles_game/requirements-0x01?urlpath=/proxy/5006/bokeh-app`](https://mybinder.org/v2/gh/riveSunder/carles_game/requirements-0x01?urlpath=/proxy/5006/bokeh-app)
```

Comment:
Update the link here to point to master, so that it will stay up to date and still work after this branch is deleted.